sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -693,6 +719,12 @@ public void reregistered(SchedulerDriver driver, Protos.MasterInfo masterInfo) {
 
     @Override
     public void resourceOffers(SchedulerDriver driver, List<Protos.Offer> offersToProcess) {
+        if (!apiServerDisabled.get() && (apiServer == null || !apiServer.isStarted())) {
+            LOGGER.info("Waiting for API Server to start ...");
+            declineOffers(driver, Collections.emptyList(), offersToProcess);
+            return;
```

Comment:
I wasn't aware Elastic took so long.  Why is that?  It's take 1.5-2 minutes for the first Task to get launched?  I don't remember that being the case.